### PR TITLE
modal component

### PR DESCRIPTION
**Pull request by preethasri**
created modal component  with live demo

### CODE REVIEW

**Review by Chakravarthi0 — COMMENTED** [review]
Hey preetha,

suggestion: use variables for colors and use rem unit for values greater than 10px.

rest looks good.

**Comment by preethasri** [comment]
> Hey preetha,
> 
> suggestion: use variables for colors and use rem unit for values greater than 10px.
> 
> rest looks good.

Thank you saran . I will definitely change it . I am now working on other stuffs . Will definitely keep this in my mind and change it  during refactoring all my components

**Comment by Chakravarthi0** [comment]
> > Hey preetha,
> > suggestion: use variables for colors and use rem unit for values greater than 10px.
> > rest looks good.
> 
> Thank you saran . I will definitely change it . I am now working on other stuffs . Will definitely keep this in my mind and change it during refactoring all my components

cool

**Review by Chakravarthi0 — APPROVED** [review]

File: modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    font-size: 1.1rem;
+    margin-bottom: 1.5rem;
+}
+
+.btn {
+    text-align: center;
+}
+
+.live-demo-button {
+    padding: 20px 20px;
```
Comment (gopalbbharadva) [inline modal/modal.css:24]: Suggestion: padding:20px , will affect the same.

File: modal/modal.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                    style="width: 608px; height: 743px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin" type="">
```
Comment (gopalbbharadva) [inline modal/modal.html:47]: Suggestion : you should avoid inline styles, it's a bad practice.

**Review by gopalbbharadva — APPROVED** [review]

File: modal/modal.html (hunk 2 above)
Comment (preethasri) [reply modal/modal.html:47]: > Suggestion : you should avoid inline styles, it's a bad practice.

will look at it gopal

File: modal/modal.css (hunk 1 above)
Comment (preethasri) [reply modal/modal.css:24]: > Suggestion: padding:20px , will affect the same.

my bad.will keep that in mind .thanks for pointing out

**Pull request merged**